docs/queries/filters.md

```diff
@@ -121,44 +121,52 @@ For full details of combining filters with boolean operators, see [Combining Fil
 - `no done date`
 - `has done date`
 - `done (before|after|on) <date>`
+- `done in this (week|month|quarter|half|year)`
```

Comment:
I forgot to mention earlier... I'm not sure where to add this, but I feel that it would be worth the documentation explaining how the `week` one selects its boundaries...

Reply:
I mean that it uses the correct 'start of week' - but I can't remember the proper vocabulary for it.